index.js

```diff
@@ -2,39 +2,48 @@
 'use strict';
 
 var path = require('path');
+var checker = require('ember-cli-version-checker');
 var HTMLBarsInlinePrecompilePlugin = require('babel-plugin-htmlbars-inline-precompile');
 
 module.exports = {
   name: 'ember-cli-htmlbars-inline-precompile',
 
-  included: function(app) {
-    this._super.included(app);
-
-    app.options.babel = app.options.babel || {};
-    app.options.babel.plugins = app.options.babel.plugins || [];
-
-    var Compiler = require(this.templateCompilerPath());
-    var PrecompileInlineHTMLBarsPlugin = HTMLBarsInlinePrecompilePlugin(Compiler.precompile);
+  parentRegistry: null,
 
-    // add the HTMLBarsInlinePrecompilePlugin to the list of plugins used by
-    // the `ember-cli-babel` addon
-    app.options.babel.plugins.push(PrecompileInlineHTMLBarsPlugin);
+  init: function() {
+    checker.assertAbove(this, '0.1.2');
```

Comment:
Yeah, makes sense. At least would give a useful error for folks on older versions.